### DISCUSSION TITLE
chosing the display model

### DISCUSSION THREAD

**Discussion by greymda** [alexanderlavrushko/BLE-HUD-navigation-ESP32#7] [General]
sorry for dumb question, absolutely new to all this. as i don't have anything Arduino related locally, will source them from China.
now, i see we kinda have a choice for display between Waveshare 14747 128x128 OLED RGB  and TTGO T-Display.
my question is: apart from display size, are they outputing the same data/info (same lane info, same arrows, etc.)?

again, sorry for dumb question. hope you reply, though.

**Comment by alexanderlavrushko** [comment]
Yes, currently it works only with these 2 displays, and data is the same: next instruction with distance to it, and current speed limit.

**Reply by greymda** [reply]
thank you for prompt reply. as i'm waiting for when you'll make a option for mirrored output, hopefully my board will arrive from China. my plan is to use it as a HUD on my car, for GPS navigation. thanks again
